web-ui/src/app/components/step/results/results.component.html

```diff
@@ -197,8 +196,12 @@ <h1 class="title" i18n>
             </p-table>
         </ng-container>
         <ng-template #warning>
-            <div *ngIf="!loading" class="notification is-warning">No results found, please try again with another
-                treebank or other xpath-query</div>
+            <div *ngIf="!loading" class="notification is-warning">
+                <ng-container *ngIf="activeFilterCount">No results found, please try again with another treebank, other
```

Comment:
For less schizophrenia (i.e., more consistency with the next condition) I suggest formulating this condition as follows:

```suggestion
                <ng-container *ngIf="activeFilterCount > 0">No results found, please try again with another treebank, other
```